src/applications/mhv/landing-page/utilities/data/index.js

```diff
@@ -18,10 +19,27 @@ const resolveToggleLink = (link, featureToggles) => {
   return { href, text, key: toggle };
 };
 
+const countUnreadMessages = folders => {
+  let unreadMessageCount = 0;
+  if (Array.isArray(folders?.data)) {
+    unreadMessageCount = folders.data.reduce((accumulator, currentFolder) => {
+      return accumulator + currentFolder.attributes?.unreadCount;
+    }, 0);
+  } else if (folders?.data?.attributes?.unreadCount > 0) {
+    unreadMessageCount = folders.data.attributes.unreadCount;
+  }
+
+  return unreadMessageCount;
+};
```

Comment:
Is the requirement here to show an indicator for an unread message in _any_ folder, or an unread message in the inbox?

Reply:
slack discussion. tl;dr, we should look at all folders.